WEB/BE/utils/bcrypt/index.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const saltRounds = Number(process.env.SALTROUNDS);
+
+const encryptedPassword = (password) => {
+  return new Promise((resolve, reject) => {
+    bcrypt.hash(password, saltRounds, (err, hash) => {
+      resolve(hash);
+    });
+  });
+};
```

Comment:
에러가 발생했을 때 reject하는 로직이 필요하지 않을까요?
```suggestion
const encryptedPassword = (password) => {
  return new Promise((resolve, reject) => {
    bcrypt.hash(password, saltRounds, (err, hash) => {
      if (err) reject(err);
      resolve(hash);
    });
  });
};
```

Reply:
와우 뺴먹은 부분을 봐주셨네요 ㅠㅠ 감사합니다 !!